
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <title>Satellite Workshop | 16th Conference on Research in Computational Molecular Biology | RECOMB 2012 </title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="shortcut icon" href="sites/all/themes/blogbuzz/favicon.ico" type="image/x-icon" />
<meta property="og:title" content="Satellite Workshop | 16th Conference on Research in Computational Molecular Biology | RECOMB 2012" />
<meta name="description" content="Satelite Workshop on the RECOMB 2012, 16th Conference on Computational Molecular Biology and Genomic Sequencing from 21-24 April in Barcelona, Spain. Focused on computational challenges arising from the extraordinary developments in high throughput technologies." />
<meta name="keywords" content="recomb,recomb 2012,crg,centre for genomic regulation,genome sequencing,genome conference 2012,satellite.html workshop,workshop" />
<meta name="dcterms.date" content="2012-02-20T00:00:00Z" />
<link rel="canonical" href="node/17.html" />
<meta property="og:url" content="http://recomb2012.crg.cat/node/17.html" />
<meta name="revisit-after" content="1 day" />
  <link type="text/css" rel="stylesheet" media="all" href="http://recomb2012.crg.cat/modules/node/node.css" />
<link type="text/css" rel="stylesheet" media="all" href="http://recomb2012.crg.cat/modules/system/defaults.css" />
<link type="text/css" rel="stylesheet" media="all" href="http://recomb2012.crg.cat/modules/system/system.css" />
<link type="text/css" rel="stylesheet" media="all" href="http://recomb2012.crg.cat/modules/system/system-menus.css" />
<link type="text/css" rel="stylesheet" media="all" href="http://recomb2012.crg.cat/modules/user/user.css" />
<link type="text/css" rel="stylesheet" media="all" href="sites/all/modules/cck/theme/content-module.css" />
<link type="text/css" rel="stylesheet" media="all" href="sites/all/modules/ckeditor/ckeditor.css" />
<link type="text/css" rel="stylesheet" media="all" href="sites/all/modules/ctools/css/ctools.css" />
<link type="text/css" rel="stylesheet" media="all" href="sites/all/modules/date/date.css" />
<link type="text/css" rel="stylesheet" media="all" href="sites/all/modules/date/date_popup/themes/datepicker.css" />
<link type="text/css" rel="stylesheet" media="all" href="sites/all/modules/date/date_popup/themes/jquery.timeentry.css" />
<link type="text/css" rel="stylesheet" media="all" href="sites/all/modules/dhtml_menu/dhtml_menu.css" />
<link type="text/css" rel="stylesheet" media="all" href="sites/all/modules/filefield/filefield.css" />
<link type="text/css" rel="stylesheet" media="all" href="sites/all/modules/panels/css/panels.css" />
<link type="text/css" rel="stylesheet" media="all" href="sites/all/modules/tagadelic/tagadelic.css" />
<link type="text/css" rel="stylesheet" media="all" href="sites/all/modules/cck/modules/fieldgroup/fieldgroup.css" />
<link type="text/css" rel="stylesheet" media="all" href="sites/all/modules/views/css/views.css" />
<link type="text/css" rel="stylesheet" media="all" href="sites/all/themes/blogbuzz/css/blogbuzz.css" />
<link type="text/css" rel="stylesheet" media="all" href="sites/all/themes/blogbuzz/css/recomb.css" />
 
  <!--[if IE 7]>
  <link type="text/css" rel="stylesheet" href="/sites/all/themes/blogbuzz/css/ie7.css" media="screen">
  <![endif]-->
  <!--[if IE 8]>
  <link type="text/css" rel="stylesheet" href="/sites/all/themes/blogbuzz/css/ie8.css" media="screen">
  <![endif]-->
  <script type="text/javascript" src="http://recomb2012.crg.cat/misc/jquery.js"></script>
<script type="text/javascript" src="http://recomb2012.crg.cat/misc/drupal.js"></script>
<script type="text/javascript" src="sites/all/modules/dhtml_menu/dhtml_menu.js"></script>
<script type="text/javascript" src="sites/all/modules/google_analytics/googleanalytics.js"></script>
<script type="text/javascript" src="sites/all/modules/panels/js/panels.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, { "basePath": "/", "dhtmlMenu": { "slide": "slide", "clone": "clone" }, "googleanalytics": { "trackOutgoing": 1, "trackMailto": 1, "trackDownload": 1, "trackDownloadExtensions": "7z|aac|arc|arj|asf|asx|avi|bin|csv|doc|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls|xml|z|zip" } });
//--><!]]>
</script>
</head>

<body class="not-front not-logged-in node-type-page one-sidebar sidebar-left page-satellite.html section-satellite.html full-node">
  <div id="page"><div id="page-inner">
        <div id="header"><div id="header-inner" class="clear-block">
              <div id="logo-title">
                      <div id="logo"><a href="index.html" title="Home" rel="home.html"><img src="sites/default/files/logo.jpg" alt="Home" id="logo-image" /></a></div>
          		  <div id="site-name-slogan">
                        		  </div>
        </div> <!-- /#logo-title -->
      	      </div></div> <!-- /#header-inner, /#header -->

          <div id="primary"><div id="primary-inner" class="clear-block">
        <ul class="menu"><li class="leaf first dhtml-menu "><a href="home.html" title="Welcome to RECOMB 2012" id="dhtml_menu-380">Home</a></li>
<li class="leaf  dhtml-menu  active-trail"><a href="satellite.html" title="Satellite Workshop | 16th Conference on Computational Molecular Biology | RECOMB 2012" id="dhtml_menu-647" class="active">Satellite</a></li>
<li class="leaf  dhtml-menu "><a href="comittees.html" title="Committees" id="dhtml_menu-544">Committees</a></li>
<li class="leaf  dhtml-menu "><a href="about-recomb-and-recomb-conference-series.html" title="About RECOMB and RECOMB Conference Series" id="dhtml_menu-545">About</a></li>
<li class="leaf  dhtml-menu "><a href="http://recomb2012.crg.cat/speakers" id="dhtml_menu-442">Speakers</a></li>
<li class="leaf  dhtml-menu "><a href="content/program.html" title="Program" id="dhtml_menu-419">Program</a></li>
<li class="leaf  dhtml-menu "><a href="content/registration.html" title="Registration" id="dhtml_menu-420">Registration</a></li>
<li class="leaf  dhtml-menu "><a href="content/papers.html" title="Papers" id="dhtml_menu-421">Papers</a></li>
<li class="leaf  dhtml-menu "><a href="content/posters.html" title="Posters" id="dhtml_menu-422">Posters</a></li>
<li class="leaf  dhtml-menu "><a href="highlights.html" title="Highlights" id="dhtml_menu-637">Highlights</a></li>
<li class="leaf  dhtml-menu "><a href="content/venue.html" title="Venue" id="dhtml_menu-423">Venue</a></li>
<li class="leaf last dhtml-menu "><a href="content/sponsors.html" title="Sponsors" id="dhtml_menu-424">Sponsors</a></li>
</ul>      </div></div>
     <!-- /#primary -->

	<div id="main">
	  <div id="main-inner">
	    <div class="clear-block">
            
          <div id="content">
		    <div id="breadcrumb">You are here: <div class="breadcrumb"><a href="index.html">Home</a></div></div>	
            <div id="content-inner">	      
		                      <div id="content-header">
                                                                                        </div> <!-- /#content-header -->
                            
<div class="node node-first node-type-page">
  <div class="node-inner clear-block">
                    <div class="content clear-block">
      <h1>
	Satellite Workshop RECOMB 2012</h1>
<p>
	<strong>SECOND ANNUAL RECOMB SATELLITE WORKSHOP ON MASSIVELY PARALLEL SEQUENCING (RECOMB-seq)</strong></p>
<p>
	<br />
	April 19-20, 2012<br />
	PRBB Building, Barcelona, Spain</p>
<p>
	Please note the venue of RECOMB-seq is NOT the same as RECOMB2012. Click <a href="http://g.co/maps/4ggyj" target="_blank">here</a> to see where the PRBB Building is located.</p>
<hr />
<h3>
	&nbsp;</h3>
<h2>
	The supplement <strong>&ldquo;Proceedings of the Second Annual RECOMB Satellite Workshop on Massively Parallel Sequencing (RECOMB-seq 2012)&rdquo; </strong>has been published today. &nbsp;The full content of the supplement can be found at <a href="http://www.biomedcentral.com/bmcbioinformatics/supplements/13/S6">http://www.biomedcentral.com/bmcbioinformatics/supplements/13/S6</a><a href="http://www.biomedcentral.com/bmcbioinformatics/supplements/13/S6 " target="_blank"> </a>.&nbsp;</h2>
<div>
	&nbsp;</div>
<hr />
<p class="rtejustify">
	<br />
	<strong>About the workshop<br />
	</strong>RECOMB-seq is a workshop held two days prior to the main conference, with a distinct paper submission process and program. Our goal is to bring together the community of scientists working on methods for massively parallel sequencing.</p>
<p class="rtejustify">
	<strong>Scope</strong><br />
	The recent revolution in sequencing technology has opened the door for myriads of new applications and bio-medical discoveries. Projects are under way to sequence thousands of individuals (the 1000 genomes project), tens of thousands of vertebrate species (the Genome10K project), and the whole microbial ecosystem that live in our bodies (the Human Microbiome project). Simultaneously, the novelty and complexity of the data has highlighted the challenges and limitations of current methods. As the technology continues to evolve and approaches its third generation, the challenges facing the community are becoming increasingly computational.</p>
<p class="rtejustify">
	<br />
	We would like to invite contributions describing new methodology to deal with all aspects of massively parallel sequencing data, including, but not limited to,</p>
<ul>
<li>
		Bio-medical applications, including cancer genomics</li>
<li>
		Discovery and genotyping of genomic variants; including SNPs, indels, and structural variants</li>
<li>
		Local and <em>de novo</em> sequence assembly</li>
<li>
		RNA sequencing, including the analysis of RNA expression and novel transcript assembly</li>
<li>
		Epigenetics, including ChIP-SEQ analysis, methylation profiling, and histone modifications</li>
<li>
		Metagenomics</li>
<li>
		Read mapping</li>
<li>
		Methods for emerging sequencing technologies, such as single-cell or single molecule real time sequencing</li>
</ul>
<hr />
<p>
	<strong>DOWNLOAD THE PROGRAM OF RECOMB-SEQ <a href="sites/default/files/RECOMB-seq&#32;PROGRAM.pdf" target="_blank">HERE</a>.</strong></p>
<hr />
<p>
	<strong>Keynote Speakers</strong></p>
<p>
	<img alt="" src="sites/default/files/zelikovsky_large.jpg" style="width: 110px; height: 138px; float: left;" /><strong><a href="http://www.cs.gsu.edu/?q=a_zelikovsky" target="_blank">&nbsp; Alex ZELIKOVSKY</a></strong>,<br />
	&nbsp; Georgia State University, Atlanta, US<br />
	&nbsp; <strong>&quot;Reconstruction of Haplotype Spectra from NGS Reads&quot;</strong></p>
<table border="0" cellpadding="0" cellspacing="0" style="width: 489px; height: 354px;">
<colgroup>
		<col /><br />
	</colgroup>
<tbody>
<tr height="287">
<td class="rtejustify" height="287" style="height:287px;width:304px;">
				In many applications of NGS, sequenced samples contain multiple distinct but sometimes very similar sequences. This talk addresses two such applications: reconstructing viral quasispecies from shotgun and amplicon 454 Life Sciences reads and transcriptome reconstruction from RNA-Seq single and paired reads.<br />
				Error-prone replication of RNA viruses with high mutation rate creates a diverse population of closely related variants known as quasispecies. By understanding the quasispecies, more effective drugs and vaccines can be manufactured as well as cost-saving metrics for infected patients implemented.&nbsp; Reconstructing the quasispecies spectrum is difficult since conserved regions in the genome can extend beyond the read length. We will compare a recently proposed approach of reconstruction from amplicon reads obtained from predefined overlapped genome regions with more developed reconstruction from shotgun reads randomly distributed along the entire genome.<br />
				Transcriptome reconstruction and quantification from RNA-Seq reads can be greatly improved by using existing partial annotation as well as fully utilizing paired reads. We describe a framework which removes reads that can be explained by existing annotation and focuses on remaining reads for discovering novel transcripts and estimating its frequency.&nbsp; We next describe an optimization approach that minimizes number of selected candidate transcripts such that mapped paired reads closely follow known insert length distribution.</td>
</tr>
</tbody>
</table>
<p>
	&nbsp;</p>
<p>
	<img alt="" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAMkAAAD/CAIAAADUh2XiAAAC0klEQVR4nO3VsQ2DQAAEQcIvgfDLoxPKoAzKwwmRJWdeIaGZCi5Y6ZYLGst1Xed5zjkX+Ic553Ecd1vruj69h1cZY9xtPb2EF/pu67lr5iW0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1S0RUVbVLRFRVtUtEVFW1R+tgX/crc1xnh6CW9zt7Xv+9NLeJVt2+62oPABau84+D/3zoAAAAAASUVORK5CYII=" style="width: 113px; height: 144px; float: left;" /><strong>&nbsp; </strong><strong><a href="http://cseweb.ucsd.edu/%7Eppevzner/" target="_blank">Pavel PEVZNER</a></strong>,<br />
	&nbsp; University of California, San Diego, US<br />
	&nbsp; <strong>&quot;SPAdes: a New Genome Assembly Algorithm and its Applications to Single-Cell Sequencing&quot;</strong></p>
<table border="0" cellpadding="0" cellspacing="0" style="width: 489px; height: 358px;">
<colgroup>
		<col /><br />
	</colgroup>
<tbody>
<tr height="287">
<td class="rtejustify" height="287" style="height:287px;width:304px;">
<p>
					The lion&#39;s share of bacteria in various environments cannot be cloned in the laboratory and thus cannot be sequenced using existing technologies. A major goal of single-cell genomics is to complement gene-centric metagenomic data with whole-genome assemblies of uncultivated organisms. Assembly of single-cell data is challenging because of highly non-uniform read coverage as well as elevated levels of sequencing errors and chimeric reads. We describe SPAdes, a new assembler for both single-cell and standard (multicell) assembly, and demonstrate that it improves on the recently released E+V-SC assembler (specialized for single-cell data) and on popular assem blers Velvet and SoapDeNovo (for multicell data). SPAdes generates single-cell assemblies, providing information about genomes of uncultivatable bacteria that vastly exceeds what may be obtained via traditional metagenomics studies.</p>
<p>
					This is a joint work with Anton Bankevich, Sergey Nurk, Dmitry Antipov, Alexey Gurevich, Mikhail Dvorkin, Alexander Kulikov, Valery Lesin, Sergey Nikolenko, Son Pham, Andrey Prjibelski, Alexey Pyshkin, Alexander Sirotkin, Nikolay Vyahhi, Glenn Tesler, and Max Alekseyev.</p>
<p>
					&nbsp;</p>
</td>
</tr>
</tbody>
</table>
<hr />
<p>
	<strong>Steering Committee</strong></p>
<ul>
<li>
		<a href="http://compbio.cs.sfu.ca/people/sahinalp.htm" target="_blank">S. Cenk Åžahinalp</a>, Simon Fraser University, Canada</li>
<li>
		<a href="http://www.cs.toronto.edu/%7Ebrudno/" target="_blank">Michael Brudno</a>, University of Toronto, Canada</li>
<li>
		<a href="http://www.bcgsc.ca/faculty/inanc-birol" target="_blank">Inanc Birol</a>, BC Genome Sciences Centre, Canada</li>
</ul>
<p>
	<strong>Program Committee</strong> <strong>Chairs</strong></p>
<ul>
<li>
		Paul Medvedev, UC San Diego, USA</li>
<li>
		Eleazar Eskin, UCLA, USA</li>
</ul>
<p>
	<strong>Program Committee</strong></p>
<ul>
<li>
		Can Alkan, University of Washington, USA</li>
<li>
		Vineet Bafna, UC San Diego, USA</li>
<li>
		Vikas Bansal, Scripps Translational Science Institute, USA</li>
<li>
		Dumitru Brinza, Life Technologies, USA</li>
<li>
		Titus Brown, Michigan State University, USA</li>
<li>
		Marc Chaisson, Pacific Biosciences, USA &nbsp;</li>
<li>
		Hamidreza Chitsaz, Wayne State University, USA</li>
<li>
		Nilgun Donmez, University of Toronto, Canada</li>
<li>
		Jason Ernst, MIT, USA</li>
<li>
		Eran Halperin, Tel Aviv University, Israel</li>
<li>
		Nils Homer, Life Technologies, USA</li>
<li>
		Fereydoun Hormozdiari, Simon Frasier University, Canada</li>
<li>
		Zam Iqbal, University of Oxford, UK</li>
<li>
		Christina Leslie, Memorial Sloan-Kettering Cancer Center, USA&nbsp;</li>
<li>
		Ion Mandoiu, University of Connecticut, USA</li>
<li>
		Gil McVean, University of Oxford, UK</li>
<li>
		Hyun Min Kang, University of Michigan, USA</li>
<li>
		William Noble, University of Washington, USA</li>
<li>
		Bogdan Pasanuic, Harvard, USA</li>
<li>
		Benedict Paten, UC Santa Cruz, USA</li>
<li>
		Ben Raphael, Brown, USA</li>
<li>
		Jared T. Simpson, University of Cambrige, Sanger Institute, UK</li>
<li>
		Suzanne Sindi, Brown, USA</li>
<li>
		Jens Stoye, Bielefeld University, Germany</li>
<li>
		Fabio Vandin, Brown, USA</li>
<li>
		Noah Zaitlen, Harvard, USA</li>
<li>
		Alex Zelikovsky, Georgia State University, USA</li>
<li>
		Daniel Zerbino, UC Santa Cruz, USA</li>
</ul>
<hr />
<p class="rtejustify">
	<strong>List of Accepted Posters:</strong></p>
<ul>
<li class="rtejustify">
		Jos&eacute; Ignacio Lucas Lled&oacute; and Mario C&aacute;ceres. Detection of chromosomal inversions with paired-end sequencing</li>
<li class="rtejustify">
		Thomas Bonfert, Gergely Csaba, Ralf Zimmer and Caroline C. Friedel. A context-based approach to identify the most likely mapping for RNA-seq experiments</li>
<li class="rtejustify">
		Ray Marin and Jiri Vanicek. Understanding the nucleation of the microRNA-mRNA pairing by using CLIP-Seq and RNA folding data</li>
<li class="rtejustify">
		S&ograve;nia Casillas, Can Alkan, Evan E Eichler and Mario C&aacute;ceres. Calling inversions from Next-Generation Sequencing Paired-End Mapping data with GRIAL</li>
<li class="rtejustify">
		S&iacute;lvia Bon&agrave;s, Josep M. Mercader and David Torrents. Empirical evaluation of different modern reference panels for imputation and their implication for Genome Wide Association Studies.</li>
<li class="rtejustify">
		Susanne Balzer, Ketil Malde, Inge Jonassen and Markus A. Grohme. Filtering duplicate reads from 454 pyrosequencing data</li>
<li class="rtejustify">
		Eva K&ouml;nig, Lars Feuerbach, Barbara Hutter, Matthias Schlesner, Qi Wang, Benedikt Brors and Thomas Lengauer. Improving loss of heterozygosity identification by tumor purity estimation</li>
<li class="rtejustify">
		Anna Ritz, Suzanne Sindi, Ali Bashir and Benjamin Raphael. A Probabilistic Method for Structural Variant Prediction from Strobe Sequencing Data</li>
<li class="rtejustify">
		Nathan Boley. Integrative De Novo Transcriptome Assembly in Fruit Fly</li>
<li class="rtejustify">
		Ole Schulz-Trieglaff, Elizabeth Murchison, Zemin Ning and Anthony Cox. The devil is in the detail: mining and annotating genomic variants in the Tasmanian Devil facial tumour genome</li>
<li class="rtejustify">
		Irina Khrebtukova, Ryan Kelley, Shujun Luo, Tim Hill, Patrick Lau, Jennifer Chiniquy, Kathryn Stephens, Semyon Kruglyak and Gary P Schroth. Automated workflow for RNA-Seq analysis: application and testing with various types of RNA-Seq protocols</li>
<li class="rtejustify">
		Andr&eacute; Corvelo and Tyler Alioto. The de novo Genome Assembly Assessment Server</li>
<li class="rtejustify">
		Kirstine Belling, David Flores, Daniel Elias, Jan Stenvang, Jun Wang, Nils Br&uuml;nner, Henrik Ditzel and Ramneek Gupta. Improving RNA sequencing interpretation: a case study on breast cancer cell lines</li>
<li class="rtejustify">
		Ryan M. Layer, Aaron R. Quinlan and Ira M. Hall. Coalescing discordant read mapping signatures for structural variant breakpoint detection</li>
<li class="rtejustify">
		Joanna Ortiz Alcantara, Elizabeth Gonz&aacute;lez Dur&aacute;n, Araceli Rodriguez Castillo, Fabiola Garc&eacute;s Ayala, Jos&eacute; Miguel Segura Candelas, Claudia Wong Ar&aacute;mbula, Patricia Alc&aacute;ntara P&eacute;rez, Abril Rodr&iacute;guez, Brisia Rodr&iacute;guez, Juan Carlos Del Mazo, Susana Serrano, Gisela Barrera Badillo, Irma L&oacute;pez Mart&iacute;nez, Luc&iacute;a Hern&aacute;ndez Rivas, Hugo L&oacute;pez-Gatell, Celia Alpuche Aranda and Jos&eacute; Ernesto Ram&iacute;rez Gonz&aacute;lez. Analysis of pandemic (H1N1) 2009 Influenza A virus circulating in Mexico during the 2011-2012 season by ultra-deep sequencing</li>
<li class="rtejustify">
		Gregory Faust and Ira Hall. YAHA: fast and flexible long-read alignment with optimal breakpoint detection</li>
<li class="rtejustify">
		Norbert Dojer and Piotr Jaroszy.ski. Efficient and error-tolerant sequencing read mapping</li>
<li class="rtejustify">
		John R. McPherson, Yingting Wu, Patrick Tan and Steve Rozen. Identifying Genomic Copy Number Alteration and Loss of Heterozygosity in Next-Generation Sequence Data</li>
<li class="rtejustify">
		Fabio Vandin, Hsin-Ta Wu, Eli Upfal and Ben Raphael. Algorithms to Find Mutated Pathways in Cancer</li>
<li class="rtejustify">
		Dumitru Brinza, Zheng Zhang, Eric Tsung, Charles Scafe, Onur Sakarya, Alexander Joyner, Sowmi Utiramerur, Guy Del Mistro, Fiona Hyland and Ellen Beasley. Torrent Variant Caller: It&#39;s all about Speed, Accuracy, and Long Indels</li>
<li class="rtejustify">
		Santiago Marco Sola and Paolo Ribeca. The GEM toolkit: world-class short read mapping, and more</li>
<li class="rtejustify">
		Maga Rowicka and Abhishek Mitra. Strategies for sequencing and analysis of low-diversity samples</li>
<li class="rtejustify">
		Abhishek Mitra and Maga Rowicka. Instant-Seq:- an integrated tool with web interface for fast analysis of ChIP-Seq data</li>
<li class="rtejustify">
		Nicholas Mancuso, Bassam Tork, Pavel Skums, Ion Mandoiu and Alex Zelikovsky. Poster: Quasispecies Spectrum Reconstruction using Multi-commodity Flows</li>
<li class="rtejustify">
		Aida Ouangraoua, Krister Swenson and Anne Bergeron. On the comparison of sets of alternative transcripts (Poster)</li>
<li class="rtejustify">
		Zhanyong Wang, Farhad Hormozdiari, Wen-Yun Yang, Eran Halperin and Eleazar Eskin. CNVeM: Copy Number Variation Detection Using Uncertainty of Read Mapping</li>
<li class="rtejustify">
		David G. Knowles, Maik R&ouml;der, Angelika Merkel and Roderic Guig&oacute;. GRAPE RNAseq Analysis Pipeline Environment<br />
		&nbsp;</li>
</ul>
<hr />
<p>
	<strong>List of Accepted Papers &amp; Presentations:</strong></p>
<ul>
<li class="paper">
		<span class="authors"><span>Ravi Vijaya Satya, Nela Zavaljevski and Jaques Reifman</span>. </span> <span class="title"> A new strategy to reduce allelic bias in RNA-Seq read-mapping </span></li>
<li class="paper">
		<span class="authors"><span>Faraz Hach, Ibrahim Numanagic, Can Alkan and Suleyman Cenk Sahinalp</span>. </span> <span class="title"> SCALCE: boosting Sequence Compression Algorithms using Locally Consistent Encoding </span></li>
<li class="paper">
		<span class="authors"><span>Chengxi Ye, Zhanshan Ma, Charles Cannon, Mihai Pop and Douglas Yu</span>. </span> <span class="title"> Exploiting Sparseness in de novo Genome Assembly </span></li>
<li class="paper">
		<span class="authors"><span>Roye Rozov, Eran Halperin and Ron Shamir</span>. </span> <span class="title"> MGMR: leveraging RNA-Seq population data to optimize expression estimation </span></li>
<li class="paper">
		<span class="authors"><span>Tal Efros and Eran Halperin</span>. </span> <span class="title"> Haplotype reconstruction using perfect phylogeny and sequence data </span></li>
<li class="paper">
		<span class="authors"><span>Ekaterina Khrameeva and Mikhail Gelfand</span>. </span> <span class="title"> Biases in read coverage demonstrated by interlaboratory and interplatform comparison of 117 mRNA and genome sequencing experiments </span></li>
<li class="paper">
		<span class="authors"><span>Gustavo Sacomoto, Kielbassa Janice, Rayan Chikhi, Raluca Uricaru, Pavlos Antoniou, Marie-France Sagot, Pierre Peterlongo and Vincent Lacroix</span>. </span> <span class="title"> KisSplice: de-novo calling&nbsp;&nbsp;alternative splicing events from RNA-seq data </span></li>
<li class="paper">
		<span class="authors"><span>Jin Zhang, Jiayin Wang and Yufeng Wu</span>. </span> <span class="title"> An Improved Approach for Accurate and Efficient Calling of Structural Variations with Low-coverage Sequence Data </span></li>
<li class="paper">
		<span class="authors"><span>J&oacute;n Ingi Sveinbj&ouml;rnsson and Bjarni Halldorsson</span>. </span> <span class="title"> PAIR: Polymorphic Alu Insertion Recognition </span></li>
<li class="paper">
		<span class="authors"><span>Sangwoo Kim, Paul Medvedev and Vineet Bafna</span>. </span> <span class="title"> Systematic Identification of Intersperse Duplication using Paired End Sequencing </span></li>
<li class="paper">
		<span class="authors"><span>Tobias Marschall, Ivan Costa, Stefan Canzar, Markus Bauer, Gunnar Klau, Alexander Schliep and Alexander Schoenhuth</span>. </span> <span class="title"> CLEVER: Clique-Enumerating Variant Finder </span></li>
<li class="paper">
		<span class="authors"><span>Matthew Edwards and David Gifford</span>. </span> <span class="title"> High-resolution genetic mapping with pooled sequencing </span></li>
<li class="paper">
		<span class="authors"><span>Thomas Bonfert, Gergely Csaba, Ralf Zimmer and Caroline C. Friedel</span>. </span> <span class="title"> A context-based approach to identify the most likely mapping for RNA-seq experiments </span></li>
<li class="paper">
		<span class="authors"><span>Layla Oesper, Anna Ritz, Sarah Aerni, Ryan Drebin and Ben Raphael</span>. </span> <span class="title"> Reconstructing Cancer Genome Organization </span></li>
<li class="paper">
		<span class="authors"><span>Boyko Kakaradov, Hui Yuan Xiong, Leo J. Lee, Nebojsa Jojic and Brendan Frey</span>. </span> <span class="title"> Robustly estimating percent inclusion of alternatively spliced junctions from low-coverage RNA-seq data </span></li>
</ul>
<p>
	Downloadable list available <a href="sites/default/files/RECOMB-seq&#32;2012&#32;Accepted&#32;Papers.pdf">here</a>.</p>
    </div>
      </div>
</div> <!-- /node-inner, /node -->            </div> <!-- /#content-inner -->	
          </div> <!-- /#content -->

                      <div id="sidebar-left">
              
<div class="block block-block ">
  <div class="block-inner clear-block">
    	<div class="content-wrapper"><div class="content">
      <p style="margin-top:0;">
	<a href="https://veci.eventszone.net/bcn/myCongress5/users.php?thisFormCongress=recomb2012&amp;thisFormLanguage=en" onclick="_gaq.push(['_trackEvent', 'Banner register right', window.location.pathname]);" target="_blank"><img alt="" src="sites/default/files/registernow-250px.jpg" style="width: 250px; height: 70px;" /></a><br />
	<a href="content/papers.html" target="_blank"><img alt="" src="sites/default/files/acceptedpapers-250px.jpg" style="width: 250px;" /></a><br />
	<a href="sites/default/files/RECOMB-seq&#32;2012&#32;Accepted&#32;Papers.pdf" target="_blank"><img alt="" src="sites/default/files/acceptedpapersrecombseq-250px.jpg" style="width: 250px; height: 70px; " /></a></p>
<p>	<a href="http://www.iscb.org" target="_blank"><img alt="" src="sites/default/files/ISCB-logo-affiliated-conference-250px.jpg" style="width: 250px; height: 113px;" /></a></p>
<div style="background:white;width:250px;text-align:center;padding-top:20px;padding-bottom:15px;">
	<a href="http://f1000.com/posters" target="_blank"><img alt="" src="sites/default/files/F1000Posters-Open&#32;logo_RGBweb.jpg" style="width: 221px; height: 42px;" /></a></div>
<p>
	<a href="http://f1000.com/posters" target="_blank"> </a></p>
<p>
<iframe allowfullscreen="" frameborder="0" height="315" src="http://www.youtube.com/embed/m5fx_RSuRi8" width="250px"></iframe><br />

</p>
<p>
<iframe allowfullscreen="" frameborder="0" height="315" src="http://www.youtube.com/embed/JVV3yAX3xM8" width="250px"></iframe>
</p>
    </div></div>
  </div>
</div> <!-- /block-inner, /block -->
<div class="block block-search ">
  <div class="block-inner clear-block">
    	<div class="content-wrapper"><div class="content">
      <form action="satellite.html"  accept-charset="UTF-8" method="post" id="search-block-form">
<div><div class="form-item" id="edit-search-block-form-1-wrapper">
 <input type="text" maxlength="128" name="search_block_form" id="edit-search-block-form-1" size="15" value="" title="Enter the terms you wish to search for." class="form-text" />
</div>
<span class="button"><input type="submit" name="op" id="edit-submit" value="Search"  class="form-submit" /></span>
<input type="hidden" name="form_build_id" id="form-MMjj1PnjAmCpsVfmhkhAeXaClMEwc32w4J-hxZZ1HY0" value="form-MMjj1PnjAmCpsVfmhkhAeXaClMEwc32w4J-hxZZ1HY0"  />
<input type="hidden" name="form_id" id="edit-search-block-form" value="search_block_form"  />

</div></form>
    </div></div>
  </div>
</div> <!-- /block-inner, /block -->
<div class="block block-node ">
  <div class="block-inner clear-block">
    	<div class="content-wrapper"><div class="content">
      <a href="rss.xml" class="feed-icon"><img src="http://recomb2012.crg.cat/misc/feed.png" alt="Syndicate content" title="Syndicate" width="16" height="16" /></a>    </div></div>
  </div>
</div> <!-- /block-inner, /block -->            </div> <!-- /#sidebar-left -->
          
          	    </div>

	      
	  </div>
	</div><!-- /#main-inner, /#main --> 

      
    
	<div id="closure"><div id="closure-inner"><div id="designed-by"><small><a href="http://www.antsin.com/en/" title="Drupal Theme">Designed by Antsin.com</a></small></div></div></div>
  </div></div> <!-- /#page-inner, /#page -->
  <script type="text/javascript">
<!--//--><![CDATA[//><!--
var _gaq = _gaq || [];_gaq.push(["_setAccount", "UA-26122101-2"]);_gaq.push(["_trackPageview"]);(function() {var ga = document.createElement("script");ga.type = "text/javascript";ga.async = true;ga.src = ("https:" == document.location.protocol ? "https://ssl" : "http://www") + ".google-analytics.com/ga.js";var s = document.getElementsByTagName("script")[0];s.parentNode.insertBefore(ga, s);})();
//--><!]]>
</script>
</body>
</html>
